platform/vs/docker-physyncd-vs/supervisord.conf

```diff
@@ -0,0 +1,48 @@
+[supervisord]
+logfile_maxbytes=1MB
+logfile_backups=2
+nodaemon=true
+
+[eventlistener:dependent-startup]
+command=python -m supervisord_dependent_startup
+autostart=true
+autorestart=unexpected
+startretries=0
+exitcodes=0,3
+events=PROCESS_STATE
+
+[eventlistener:supervisor-proc-exit-listener]
+command=/usr/bin/supervisor-proc-exit-listener --container-name physyncd
+events=PROCESS_STATE_EXITED
+autostart=true
+autorestart=unexpected
+
+[program:rsyslogd]
+command=/usr/sbin/rsyslogd -n -iNONE
+priority=1
+autostart=false
+autorestart=unexpected
+stdout_logfile=syslog
+stderr_logfile=syslog
+dependent_startup=true
+
+[program:start]
+command=/usr/bin/start.sh
+priority=2
+autostart=false
+autorestart=false
+startsecs=0
+stdout_logfile=syslog
+stderr_logfile=syslog
+dependent_startup=true
+dependent_startup_wait_for=rsyslogd:running
+
+[program:syncd]
```

Comment:
Do we want to refer to this program as "syncd" or "physyncd"?

Reply:
I think syncd is correct here. syncd is the name of the binary being run inside the docker. It truly is a syncd in all respects except it links with a different backend to access phy functionality.  To me, going into any of these dockers and seeing syncd in ps -eaf output is a nice consistency.

If still concerns you, maybe bounce it off of @kcudnik  he might have strong feelings either way. 

Note, I'd have to take a look also to see if such a naming change has impacts in other previously merged pulls, like sonic-sairedis, if that is the case, we already have staged those in a 202006 branch and not sure how that would complicate things. Maybe best to file an issue in that case if you want to execute a name change, and do it on master and go with what is already in the staging branch.